packages/manager/apps/key-management-service/src/index.tsx

```diff
@@ -3,14 +3,17 @@ import ReactDOM from 'react-dom/client';
 import {
   ShellProvider,
   initShellContext,
-  initI18n,
 } from '@ovh-ux/manager-react-shell-client';
 import App from './App';
+import initI18n from './i18n';
```

Comment:
using initI18n from @ovh-ux/manager-react-shell-client doesn't work ?
Or did you just revert a change without noticing ?

Reply:
It didn't work for translation on manager-components like `GuideButton`, in the other way translation wasn't apply so I took example with zimbra PR which is also using the `GuideButton`

Reply:
What is the difference between them ?
We have to fix the shared configuration instead of duplicating it on every project